### PR TITLE
fix(ci): add feg services and alphabetize semantics.yml

### PR DESCRIPTION
**Pull request by uri200**
Signed-off-by: Oriol Batalla <obatalla@fb.com>

## Summary

Removed duplicates (agw), alphabetize list and added feg services

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by tmdzk — APPROVED** [review]

File: .github/semantic.yml [hunk 1]
```diff
@@ -15,35 +15,48 @@ titleAndCommits: false
...
-  - xwf
-  - agw
-  - cloud
+  - subscriberdb
+  - swx_proxy
+  - td-agent-bit
   - upf
-  - smf
-  - amf
+  - xwf
```
Comment (uri200) [inline .github/semantic.yml:59]: lets keep this list in order to avoid duplicates (like agw)

**Review by pshelar — APPROVED** [review]
Thanks

File: .github/semantic.yml [hunk 2]
```diff
@@ -15,35 +15,48 @@ titleAndCommits: false
...
   - monitord
+  - nms
+  - orc8r
+  - pipelined
+  - policydb
+  - radiusd
   - redirectd
+  - redis
+  - s6a_proxy
+  - s8_porxy
```
Comment (ulaskozat) [inline .github/semantic.yml:49]: typo here!!

Comment (uri200) [reply .github/semantic.yml:49]: fixed here https://github.com/magma/magma/pull/8002
thanks!

**Pull request merged**